modal/app.py

```diff
@@ -1047,6 +1042,13 @@ async def _logs(self, client: Optional[_Client] = None) -> AsyncGenerator[str, N
                     if log.data:
                         yield log.data
 
+    @classmethod
+    def container_app(cls) -> Optional["_App"]:
```

Comment:
Is this something we want to be public API? Is it relevant to users in any way?

Reply:
Yeah that was my main question. I think it's nice to have a way to get the current app inside containers but I can't think of a super critical use case. Happy to prefix it by underscore if you prefer?

Reply:
I think there are a few different object types where we'd want to support "get the current instance of this object" and we might to use a consistent method name for doing that.

Reply:
In this case it's specific to containers, but yeah I agree with you. Lmk if you have strong feelings about this naming. I'll merge the PR now, but it's easy to fix as a follow up. Either way it's pretty easy to rename it later and just have a deprecation path.

Reply:
Generally speaking I guess I would default to keeping things private as it's much less pain to change that later than it is the other way.

> In this case it's specific to containers

i think that' probably true for all the modal objects? e.g. i am thinking it could apply to:

- `App`
- `Function`
- `Container`
- `FunctionCall`

all of which you might want a handle to an object representing the "current" instance. But in all cases that would only be relevant when you're running in a Modal container. But extending the pattern to `Container.container_container` will be a little sad.